src/core/datatypes/trees/Clade.cpp

```diff
@@ -125,6 +127,11 @@ bool Clade::operator==(const Clade &c) const
         return false;
     }
     
+    if ( c.clade_name != clade_name )
+    {
+        return false;
+    }
+    
```

Comment:
Hmm... do we want clades with the same members but different names to be unequal?

Reply:
My intuition says "no"...

Reply:
well, it really depends in which setting this is used. But I guess we never have two differently named but otherwise identical clades which we wanted to compare.

I've commented out the code (with a comment)...